src/plugins/timeConductor/ConductorAxis.vue

```diff
@@ -225,6 +234,63 @@ export default {
             if (this.$refs.axisHolder.clientWidth !== this.width) {
                 this.width = this.$refs.axisHolder.clientWidth;
                 this.setScale();
+                this.destroyBrush();
+                this.initBrush();
+                this.createBrush();
+            }
+        },
+        initBrush() {
+            // brush extent y coordinate starts at -1 to prevent underlying layer cursor to show
+            this.brush = d3Brush.brushX()
+                .extent([[0, -1], [this.width, this.height]])
+                .on("start", this.brushStart)
+                .on('brush', this.brushed)
+                .on("end", this.brushEnd);
+        },
+        createBrush() {
+            if (!this.isFixed) {
+                return;
+            }
+
+            this.svg = this.svg || d3Selection.select('svg');
+            if (!this.svg) {
+                return;
+            }
+
+            this.svg.append("g")
+                .attr("class", "brush")
+                .call(this.brush);
+        },
+        brushStart () {
```

Comment:
Can we remove these handlers, or do we need no-ops for some reason?

Reply:
fixed in 694255db6